### DISCUSSION TITLE
Welcome to DJL Discussions!

### DISCUSSION THREAD

**Discussion by zachgk** [deepjavalibrary/djl#412] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with DJL.

**Comment by kimim** [comment]
I'm working in an automotive manufacturing company. We have a lot of data to analysis. 
I'm fan of lisp, which is flexible and interesting. I use JVM based lisp - clojure - to write some learning programs. 
Over DJL, I create a clojure wrap [clj-djl](https://github.com/kimim/clj-djl) to deal with our data problems.

**Comment by zachgk** [comment]
Hi everyone!

I am Zach, one of the developers from AWS contributing to DJL. I'm most interested by the training side of the framework and trying to make training and understanding deep learning as easy as possible.

**Comment by lanking520** [comment]
Hello everyone!

I am Qing, one of the DJL developers. I love interesting DL projects and trying to make them accesssible to all developers. Like this undying flappybird using DJL:

![](https://raw.githubusercontent.com/kingyuluk/RL-FlappyBird/master/bird.gif)

**Comment by luhuang89** [comment]
I'm Lu. I'm the PM for DJL. I'd love to work with everyone in the community to evangelize DJL and make our DJL useful to more people! Please feel free to reach out to me if you have ideas for blogposts/meetups/other presentations. One cool project I'm also working on is to create an online learning course on DJL, please reach out if you'd like to participate in content creation for the online DJL class :-)

**Comment by sami-badawi** [comment]
I am Sami, a Scala programmer. I just stumbled upon DJL today and wonder if it makes sense to make an idiomatic Scala wrapper around DJL. Might be useful for Spark programs.

**Reply by luhuang89** [reply]
That will be awesome, Sami!

**Reply by lanking520** [reply]
That's awesome, try to start looking at some DJL spark application here: https://github.com/aws-samples/djl-demo/tree/master/spark. 

Also, this Scala notebook can work pretty well as well: https://github.com/aws-samples/djl-demo/blob/master/spark/notebook/Image_Classification_Spark.ipynb. Let me know if there is anything I can help with.

**Reply by sami-badawi** [reply]
I started a new GitHub https://github.com/sami-badawi/scaladl, all I have done so far is port ObjectDetection.java to ObjectDetection.scala.

I did run into this error when trying to access TensorFlow model:

```
[error] (run-main-0) org.tensorflow.exceptions.TensorFlowException: Could not find meta graph def matching supplied tags: { serve }. To inspect available tag-sets in the SavedModel, please use the SavedModel CLI: `saved_model_cli`
[error] org.tensorflow.exceptions.TensorFlowException: Could not find meta graph def matching supplied tags: { serve }. To inspect available tag-sets in the SavedModel, please use the SavedModel CLI: `saved_model_cli`
[error] 	at org.tensorflow.internal.c_api.AbstractTF_Status.throwExceptionIfNotOK(AbstractTF_Status.java:101)
[error] 	at org.tensorflow.SavedModelBundle.load(SavedModelBundle.java:206)
```

**Comment by kevinau** [comment]
Hi.  I'm a Java/web developer in Australia.

My goal is to try and sort thousands of photos of my children and grandchildren and, out of interest, want to use DJL and face recognition to do it.

I have the basic demos working.  I now have to put the pieces together.

Cheers from down under.
